vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,176 @@
-# Entando 6.x Release Notes 
+
+# Entando 7.0 Release Notes
+
+## General
+
+Entando version 7.0 is a major release. It includes significant changes to the K8s infrastructure provisioning mechanism that provide better security, improved reliability, a simplified deployment process and an extensibility mechanism based on the new capability provisioning workflow. Entando 7 introduces the [Entando Hub](../getting-started/landing-page.md#entando-hub), an updated compatibility matrix (e.g. Keycloak 15, PostgreSQL 14), development stack upgrades (e.g. JHipster 7.2) and several consolidation changes.
```

Comment:
@jyunmitch "extensibility" is correct here (it's a computing term)

@nshaw if this remains as one paragraph i agree with jinah; are we waiting on additional edits to the intro?

Reply:
+1 on extensibility. It's a compound word rather than an adjective-noun.
Go ahead and improve the paragraph. A high level summary of Entando 7 has been requested but I don't have an ETA.